Documentation/ceph-teardown.md

```diff
@@ -59,6 +59,19 @@ In the future this step will not be necessary when we build on the K8s local sto
 
 If you modified the demo settings, additional cleanup is up to you for devices, host paths, etc.
 
+Disks on nodes used by Rook for osds can be reset to a usable state with the following methods:
+```sh
+#!/usr/bin/env bash
+# Zap the disks to a fresh, usable state (zap-all is important, b/c MBR has to be clean)
+# This assumes that the disks used were a subset of /dev/sdb through /dev/sdzz.
+# You will likely need to alter this for your environment.
+ls /dev/vd{{b..z},{a..z}{a..z}} 2>/dev/null | xargs -I% -- sgdisk --zap-all %
```

Comment:
to simplify the readability of the documentation, what if we modify so that we're cleaning one disk? Everyone would have to modify it for their own set of disks anyway.

Reply:
Good idea. Update incoming.